client/payment-details/dispute-details/dispute-steps.tsx

```diff
@@ -0,0 +1,189 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { dateI18n } from '@wordpress/date';
+import moment from 'moment';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import type { Dispute } from 'wcpay/types/disputes';
+import { ChargeBillingDetails } from 'wcpay/types/charges';
+import { formatExplicitCurrency } from 'utils/currency';
+import { ClickTooltip } from 'wcpay/components/tooltip';
+
+interface Props {
+	dispute: Dispute;
+	customer: ChargeBillingDetails | null;
+	chargeCreated: number;
+	daysRemaining: number;
+}
+
+const DisputeSteps: React.FC< Props > = ( {
+	dispute,
+	customer,
+	chargeCreated,
+	daysRemaining,
+} ) => {
+	let emailLink;
+	if ( customer?.email ) {
+		const chargeDate = dateI18n(
+			'Y-m-d',
+			moment( chargeCreated * 1000 ).toISOString()
+		);
+		const disputeDate = dateI18n(
+			'Y-m-d',
+			moment( dispute.created * 1000 ).toISOString()
+		);
+		const emailSubject = `Problem with your purchase from ${ wcpaySettings.storeName } on ${ chargeDate }?`;
+		const emailBody =
+			`Hello ${ customer?.name }\n\n` +
+			`We noticed that on ${ disputeDate }, you disputed a ${ formatExplicitCurrency(
+				dispute.amount,
+				dispute.currency
+			) } from ${ chargeDate }. We wanted to contact you to make sure everything was all right with your purchase and see if there's anything else we can do to resolve any problems you might have had.\n\n` +
+			`Alternatively, if the dispute was a mistake, you could easily withdraw it by calling the number on the back of your card. Thank you so much - we appreciate your business and look forward to working with you.`;
+
+		emailLink = `mailto:${ customer.email }?subject=${ encodeURIComponent(
+			emailSubject
+		) }&body=${ encodeURIComponent( emailBody ) }`;
+	}
+
+	const respondByDate = dispute.evidence_details?.due_by
+		? dateI18n(
+				'M j, Y, g:ia',
+				moment( dispute.evidence_details?.due_by * 1000 ).toISOString()
+		  )
+		: '–';
+
+	return (
+		<div className="dispute-steps">
+			<div className="dispute-steps__header">
+				{ __( 'Steps to resolve:', 'woocommercts' ) }
+			</div>
+			<ol className="dispute-steps__steps">
+				<li>
+					{
+						// TODO: add link to the issuer evidence files link. See https://github.com/Automattic/woocommerce-payments/pull/7192.
```

Comment:
This will be resolved later in https://github.com/Automattic/woocommerce-payments/issues/7156.